crates/services/compression/src/errors.rs

```diff
@@ -0,0 +1,21 @@
+use thiserror::Error;
```

Comment:
Nit: I believe thiserror doesn't work with no-std. Don't think we need it for now, but if we were to pivot to run compression in a zkvm in the future it's a bit more convenient if we derive `derive_more::Error` instead.

Reply:
we will never mount the service inside a zkvm. there's no need to make this no-std compatible yet